bundles/org.openhab.binding.modbus.e3dc/src/main/resources/ESH-INF/thing/info-channel-groups.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-group-type id="info-values">
+		<label>Information</label>
```

Comment:
These channels look more like properties of a thing/bridge? Maybe this was discussed before, but is there a reason to create channels instead of properties?

Reply:
There was one remark regarding properties and as result the implementation contains a "low speed" update of these registers while Data is updated frequently.
Several enrties will change due to remote updates like Modbus Firmware Version, Supported Registers and Firmware. After the Device Installation the Serial Number wasn't set and was updated remotely. Also the Modbus-ID can change because you can operate the device either in Sunspec Mode or in the Manufacturers own Mode.

In addition I would like to have these values in my UI. It's quite handy to check currently installed Software Versions for Support. With Properties I have to change to PaperUI, search the device and check the properties. On Smartphone I didn't find a possibility to check properties yet.